<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="DiconnectButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>DISCONNECT ALL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
